src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -300,6 +300,14 @@ def _validate_list(field_content: list, is_tensor_field: bool) -> None:
 
         return
 
+    @classmethod
+    def validate_multimodal_field_name(cls, multimodal_field_name:str) -> None:
```

Comment:
You shouldn't add public methods outside the abstraction's interface (`VespaIndex` class). This applies to `validate_multimodal_field_name` and `validate_field_name`. Otherwise the abstraction becomes unusable (see Liskov substitution principle). Also have you noticed the field name validation code we already have in `marqo_index.py`?